### DISCUSSION TITLE
Missing implementations for assertions on Windows

### DISCUSSION THREAD

**Discussion by calebchalmers** [modm-io/modm#1200] [Q&A]
When building modm for a project, I am getting the following error/warning:
```
[ERROR] user.modm.platform.core: Assertions are not fully implemented!
```

This is coming from `modm/src/modm/platform/core/hosted/module.lb` when the target is `hosted-windows`. Could anyone provide info about which parts aren't implemented and what needs to be done there? I'm not familiar with the modm assertion code, but I'm willing to take a look!

**Comment by salkinium** [comment]
Specifically how to place function pointer into custom linker sections and then access them: https://github.com/modm-io/modm/blob/develop/src/modm/platform/core/cortex/assert.cpp.in#L35.
I did some research but didn't find an immediate answer for Windows.
